src/main/java/Dice.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Random;
+
+public class Dice {
+
+    private Random random;
+
+    public Dice(Random random) {
+        this.random = random;
+    }
+
+    public int rollDice(int numberOfDice) {
+
```

Comment:
It appears that the suggestion of chaning the funciton name to rollDice was already changed from roll_dice.